scripts/build_mod_ic.py

```diff
@@ -990,6 +990,28 @@
         },
     },
 
+    # VQFN-32 from BQ40Z60
+    # IPC-7351B: QFN26P500X500X100-33N
+    "QFN-32-EP-TI": {
+        "rows": 4,
+        "pins": 32,
+        "pin_pitch": 0.5,
+        "row_pitch": 4.7,
+        "pad_shape": (0.6, 0.25),
```

Comment:
From the inset on pg46 of the datasheet, the pads are 0.85mm long, not 0.60? and 0.28mm wide not 0.25?